parquet-hadoop/src/main/java/org/apache/parquet/hadoop/util/HadoopStreams.java

```diff
@@ -77,16 +78,17 @@ public static SeekableInputStream wrap(FSDataInputStream stream) {
    * @param stream stream to probe
    * @return A H2SeekableInputStream to access, or H1SeekableInputStream if the stream is not seekable
```

Comment:
Documentation should also be updated

Reply:
@rathinb-db Could you resolve this comment? Thanks!